include/SFML/System/Angle.inl

```diff
@@ -34,16 +34,14 @@ namespace sf
 {
 namespace priv
 {
-constexpr float pi = 3.141592654f;
+constexpr float pi  = 3.141592654f;
```

Comment:
I wanted to suggest `std::numbers::pi_v` but I guess we are not C++20.

Reply:
https://github.com/search?q=repo%3ASFML%2FSFML%20cxx_std_17&type=code

Alas still using C++17 since C++20 support isn't sufficient in the compilers we're targeting.